rules/coding-style/tests/Rector/Use_/UseFunction/Source/functions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Deployer;
+
+function set($name, $value)
+{
+
+}
```

Comment:
Just a tip for your project: this is not included/autoloaded. 
It must be in `composer.json` > `files`

Without that, functions are invisible :)

Reply:
My project uses deployer and it has this in composer.json https://github.com/deployphp/deployer/blob/master/composer.json#L33

I was just trying to create some minimal reprodution of the issue.

Reply:
The same must happen here too